maintainers/maintainer-list.nix

```diff
@@ -5145,6 +5145,12 @@
     githubId = 15371828;
     name = "Hugo Lageneste";
   };
+  hykilpikonna = {
```

Comment:
Please, add yourself to maintainer list as separate commit with message: `maintainers: add hykilpikonna`.

Reply:
I'm sorry. I just separated the commits and force pushed again.